data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,115 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"],
+      "description": "Unique identifier for the object."
+    },
+    "ods:type": {
+      "type": "string",
+      "examples": ["https://doi.org/21.T11148/2037de437c80264ccbce"],
+      "description": "This is the handle of the type of the object."
+    },
+    "ods:version": {
+      "type": "integer",
+      "examples": [1],
+      "description": "The version of the object."
+    },
+    "dcterms:creator": {
+      "type": "object",
+      "properties": {
+        "ods:type": {
+          "type": "string",
+          "examples": ["foaf:Person"],
+          "description": "Object containing information on who created the object.",
+          "enum": [
+            "foaf:Person",
+            "foaf:Organisation",
+            "oa:SoftwareAgent"
+          ]
+        },
+        "foaf:name": {
+          "type": "string",
+          "examples": ["My Name"],
+          "description": "Name of the creator."
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "ltc:ObjectGroup": {
+      "type": "object",
+      "properties": {
+        "ltc:conditionsOfAccess": {
+          "type": "boolean",
+          "examples": ["Open to the public"],
+          "description": "Information about who can access the collection being described or an indication of its security status."
+        },
+        "ltc:collectionName": {
+          "type": "string",
+          "examples": ["Arctiacaja1991Netherlands"],
+          "description": "A short title that summarizes the collection objects contained within the ObjectGroup."
+        },
+        "ltc:description": {
+          "type": "string",
+          "examples": ["A virtual collection representing all butterfly specimens in South Holland"],
+          "description": "A free text description or narrative about the collection."
+        },
+        "ltc:isKnownToContainTypes": {
+          "type": "array",
+          "description": "Flag property to indicate that the collection is known to include type specimens. `true` - this collection contains types; `false` - this collections does not contain types.",
+          "items": {
+            "type": "boolean",
+            "examples": ["true"],
+            "description": "Type known to be contained."
+          }
+        }
+      },
+      "description": "An intentionally grouped set of objects with one or more common characteristics."
+    },
+    "ltc:MeasurementOrFact": {
```

Comment:
How do these terms relate back to the concept of virtual collections? What kind of measurement did you have in mind?

Reply:
number of object was one of examples given by ltc.